trlx/sweep.py

```diff
@@ -86,11 +91,12 @@ def tune_function(train_function, param_space: dict, tune_config: dict, resource
         "gpu": args.num_gpus,
     }
 
+    # convert a nested path to script to a module path
```

Comment:
We need to print a warning saying that the user is about to execute arbitrary code.